package.json

```diff
@@ -2,6 +2,9 @@
   "name": "amazon-chime-sdk-js",
   "version": "1.22.0",
   "description": "Amazon Chime SDK for JavaScript",
+  "publishConfig": {
+    "tag": "release-1.x"
```

Comment:
So when installing through NPM if we do not specify anything then the latest tagged version from the release-1.x branch will be installed right?

Reply:
So this tag is for amazon-chime-sdk-js npm package. If we do not specify the tag, when we release for 1.x, it will get latest tag so anyone who do npm update amazon-chime-sdk-js will pull in this version instead of the latest 2.x version.

Reply:
Yup yup 👍